api/v1alpha1/juicefsruntime_types.go

```diff
@@ -164,6 +164,15 @@ type JuiceFSFuseSpec struct {
 	// +optional
 	CleanPolicy FuseCleanPolicy `json:"cleanPolicy,omitempty"`
 
+	// UpdateStrategy decides when to update Fuse pods.
+	// Currently Fluid supports two UpdateStrategy: OnDelete and OnIdle
+	// OnDelete update fuse pod by native daemonset once the fuse pod on some node is deleted
+	// OnIdle update fuse pod once the fuse pod on some node is in idle
+	// Defaults to OnDelete
+	// +kubebuilder:validation:Enum=OnDelete;"";OnIdle
+	// +optional
+	UpdateStrategy FuseUpdateStrategy `json:"updateStrategy,omitempty"`
```

Comment:
The implementation of `upgradeStrategy` is just the same as `cleanPolicy`. Why not merge them? so that users will not understand another parameter.